### PR TITLE
chore(common/resources): Update langtags.json

### PR DESCRIPTION
**Pull request by darcywong00**
This doesn't address the GitHub Action portion of #6591, but manually updates langtags.json from the API.
Also updates the URL to the "official schema".

## User Testing
Setup
Install the PR build of "Keyman Developer"

* **TEST_DEVELOPER** - Test language picker in Keyman Developer
1. In Keyman Developer, start a new blank project
2. In the "Languages" section, click <kbd>Add...</kbd> to bring up the language picker dialog
3. In the "Select BCP 47 Tag" dialog, select "Language tag" and enter `uta`.
4. Verify the "Language name" appears as **Itang** (Previous versions of langtags.json used the language name **Otank**).

### CODE REVIEW

**Comment by darcywong00** [comment]
Ooh, I didn't realize this would flag a Linux unit test fail (test_canonical_language_code_utils).

Working on addressing it...

File: linux/keyman-config/tests/test_canonical_language_code_utils.py [hunk 1]
```diff
@@ -21,8 +21,8 @@ def test_FindBestTag_ValidCases(self):
             { 'expected': 'sqt-Arab-YE', 'tags': ['sqt', 'sqt-YE', 'sqt-Arab'] },
             { 'expected': 'sqt-Latn-YE', 'tags': ['sqt-Latn', 'sqt-Latn-YE'] },
 
-            { 'expected': 'sa-Latn', 'tags': ['sa-Latn'] },
-            { 'expected': 'hi-Latn', 'tags': ['hi-Latn'] },
+            { 'expected': 'sa-Latn', 'tags': ['sa-Latn'] }, # Should sa-Latn go away?
+            { 'expected': 'hi-Latn-IN', 'tags': ['hi-Latn'] },
```
Comment (darcywong00) [inline linux/keyman-config/tests/test_canonical_language_code_utils.py:25]: From the updated langtags.json ll. 44534-44547
```
    {
        "full": "hi-Latn-IN",
        "iana": [ "Hindi" ],
        "iso639_3": "hin",
        "name": "Hindi",
        "names": [ "Dakhini", "Hindi-Urdu", "Hindustani", "Modern Standard Hindi" ],
        "region": "IN",
        "regionname": "India",
        "regions": [ "AE", "AU", "BD", "BT", "BZ", "CA", "DE", "DJ", "FI", "FJ", "GB", "GQ", "GY", "KE", "LK", "MU", "NP", "NZ", "PH", "PR", "RU", "SG", "SX", "TH", "US", "YE", "ZA", "ZM" ],
        "script": "Latn",
        "sldr": false,
        "tag": "hi-Latn",
        "windows": "hi-Latn"
    },
```

File: windows/src/test/unit-tests/standards-data/Keyman.Test.System.Standards.LangTagsRegistryTest.pas [hunk 2]
```diff
@@ -61,7 +61,7 @@ procedure TLangTagsTest.TestSomeTags;
   // All Tags
 
   Assert.AreEqual('arx', TLangTagsMap.AllTags['arx-BR']);
-  Assert.AreEqual('bfi-x-stokoe', TLangTagsMap.AllTags['sgn-bfi-GB-x-stokoe']);
+  Assert.AreEqual('bfi-Zzzz-x-stokoe', TLangTagsMap.AllTags['sgn-bfi-GB-x-stokoe']);
```
Comment (darcywong00) [inline windows/src/test/unit-tests/standards-data/Keyman.Test.System.Standards.LangTagsRegistryTest.pas:64]: From the updated langtags.json ll. 11324-11337
```
    {
        "full": "bfi-Zzzz-GB-x-stokoe",
        "iana": [ "British Sign Language" ],
        "iso639_3": "bfi",
        "name": "British Sign Language",
        "names": [ "BSL" ],
        "nophonvars": true,
        "region": "GB",
        "regionname": "United Kingdom",
        "script": "Zzzz",
        "sldr": false,
        "tag": "bfi-Zzzz-x-stokoe",
        "tags": [ "bfi-GB-x-stokoe", "sgn-bfi-GB-x-stokoe" ],
        "windows": "bfi-Zzzz-x-stokoe"
    },
```

File: linux/keyman-config/tests/test_canonical_language_code_utils.py (hunk 1 above)
Comment (rc-swag) [inline linux/keyman-config/tests/test_canonical_language_code_utils.py:25]: Should this be 
`{ 'expected': 'hi-Latn-IN', 'tags': ['hi-Latn', 'hi-Latn-IN'] }`

**Comment by bharanidharanj** [comment]
* **TEST_DEVELOPER (FAILED):** Tested this in Keyman Developer 15.0.244 as per the instructions and I noticed that the Language name is still showing 'Otank' instead of 'Itang'.

**Comment by bharanidharanj** [comment]
* **TEST_DEVELOPER (FAILED):** notes

**Comment by bharanidharanj** [comment]
> @bharanidharanj - I've updated to Setup step to specify
> 
> > Install the PR build of "Keyman Developer"
> 
> @keymanapp-test-bot retest

@darcywong00 Okay, Thanks. I will retest it.

File: linux/keyman-config/tests/test_canonical_language_code_utils.py (hunk 1 above)
Comment (darcywong00) [reply linux/keyman-config/tests/test_canonical_language_code_utils.py:25]: The other tests cases seem to combine the `"tag"` and `"tags"` fields. In this case, there's only
```
  "tag": "hi-Latn",
```

so maybe that's all we test?

**Comment by bharanidharanj** [comment]
* **TEST_DEVELOPER (FAILED):** Tested this as per the instructions and it seems to be not working as expected.

File: linux/keyman-config/tests/test_canonical_language_code_utils.py (hunk 1 above)
Comment (mcdurdin) [reply linux/keyman-config/tests/test_canonical_language_code_utils.py:25]: I don't understand why you've added `-IN` here?

File: resources/standards-data/langtags/readme.md [hunk 3]
```diff
@@ -11,5 +11,5 @@ This file is intended to replace alltags, ethnologue, and iso639-3 data files in
 
 * Source repository: https://github.com/silnrsi/langtags/
 * Documentation: https://github.com/silnrsi/langtags/blob/master/doc/langtags.md
-* Unofficial schema: https://gist.github.com/mcdurdin/eff65bb2d77034525120d48cbe280d30
+* Offical schema: https://github.com/silnrsi/langtags/blob/master/source/langtags_schema.json
```
Comment (mcdurdin) [inline resources/standards-data/langtags/readme.md:14]:
```suggestion
* Official schema: https://github.com/silnrsi/langtags/blob/master/source/langtags_schema.json
```

File: linux/keyman-config/tests/test_canonical_language_code_utils.py (hunk 1 above)
Comment (darcywong00) [reply linux/keyman-config/tests/test_canonical_language_code_utils.py:25]: langtags.json has a new entry for `hi-Latn-IN`. See
https://github.com/keymanapp/keyman/pull/6594#discussion_r868628392

Comment (rc-swag) [reply linux/keyman-config/tests/test_canonical_language_code_utils.py:25]: I need to read up on the lang tags and the code this was just a comment observing the tests vs the json file of other tags..  I think it should test that it matches both. I see what you say about the tag and tags fileds, but if you look at `sqt-Latn-YE` it only has `sqt-Latn` in the tag, but tests for ``sqt-Latn-YE` also.

Comment (mcdurdin) [reply linux/keyman-config/tests/test_canonical_language_code_utils.py:25]: I looked a bit deeper. `hi-Latn` is a new block in this langtags.json (it is not present in the beta branch version at all). So we need to understand why this test was present -- perhaps we should test with a different script?

Comment (darcywong00) [reply linux/keyman-config/tests/test_canonical_language_code_utils.py:25]: afaict, the unit test was introduced in #3545 to address this reported keyboard issue involving `sa-Latn` and `hi-Latn`

https://github.com/keymanapp/keyboards/pull/753#issuecomment-503832382

**Comment by darcywong00** [comment]
In discussing this PR with @mcdurdin we've decided to abandon this PR.
1. Keyman Developer is still using subtag registry instead of langtags.json. So that's why the updated language name in the user test isn't seen.
2. It'll take a major epoch in a future release to improve BCP 47 support across all the platforms.

**Pull request closed without merging**